lowkiq.gemspec

```diff
@@ -34,5 +34,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rspec-mocks", "~> 3.8"
   spec.add_development_dependency "rack-test", "~> 1.1"
-  spec.add_development_dependency "pry-byebug", "~> 3.9.0"
+  spec.add_development_dependency "pry-byebug", "~> 3.10.1"
```

Comment:
I think we should use any version of `pry-byebug`.

Reply:
@darkleaf can we have `spec.add_development_dependency "pry-byebug", ">= 3.9.0"` ? because ruby 3.2.0 requires pry-byebug 3.10.

Reply:
add ruby 3.2 to the matrix

Reply:
done !